src/js/hooks/cycleCount/useInventoryTransactionsTabExport.jsx

```diff
@@ -0,0 +1,66 @@
+import _ from 'lodash';
+import { useSelector } from 'react-redux';
+import { getCurrentLocationId } from 'selectors';
+
+import { INVENTORY_TRANSACTIONS_SUMMARY_CSV } from 'api/urls';
+import useSpinner from 'hooks/useSpinner';
+import dateWithoutTimeZone from 'utils/dateUtils';
+import exportFileFromAPI from 'utils/file-download-util';
+
+/**
+ * Hook to manage export actions for the Inventory Transactions tab in Cycle Count reporting.
+ *
+ * @param {string} startDate - The start date for the report (ISO string).
+ * @param {string} endDate - The end date for the report (ISO string).
+ * @param {Product[]} [products] - Optional list of products to include.
+ * @returns {{ actions: { label: string, defaultLabel: string, onClick: Function }[] }}
+ */
+const useInventoryTransactionsTabExport = ({
+  startDate,
+  endDate,
+  products,
+}) => {
+  const spinner = useSpinner();
+
+  const currentLocationId = useSelector(getCurrentLocationId);
+
+  const exportInventoryTransactionsReport = async (filters) => {
+    spinner.show();
+    await exportFileFromAPI({
+      url: INVENTORY_TRANSACTIONS_SUMMARY_CSV,
+      params: _.omit({
+        endDate: dateWithoutTimeZone({
+          date: filters.endDate,
+        }),
+        startDate: dateWithoutTimeZone({
+          date: filters.startDate,
+        }),
+        products: filters.products ? filters.products?.map((product) => product.id) : undefined,
```

Comment:
@alannadolny I mostly copied your code but I had to change this line to get the export to work. Was it working for you without this change on the other report?